aptos/contracts/sources/pyth.move

```diff
@@ -279,8 +286,10 @@ module pyth::pyth {
     /// Please refer to the documentation at https://docs.pyth.network/consumers/best-practices for
     /// how to how this price safely.
     /// 
-    /// Important: it is recommended to call update_price_feeds() to update the cached price
-    /// before calling this function, as get_price() will abort if the cached price is older 
+    /// Important: Pyth uses an on-demand update model, where consumers need to update the 
+    /// cached prices before using them. Please read more about this at https://docs.pyth.network/consume-data/on-demand.
+    /// It is therefore recommended to call update_price_feeds() to update the cached price
+    /// before calling get_price(), as get_price() will abort if the cached price is older 
```

Comment:
"It is therefore recommended" is indirect. How about "

"Pyth uses an on-demand update model where callers must update the cached price feeds before reading them. Please see ... This function is likely to abort unless you call update_price_feeds() to update the cached price feeds beforehand."